### PR TITLE
DDF-3352 updated configuring data management docs

### PR DESCRIPTION
**Pull request by ricklarsen**
**Port to `master` of PR https://github.com/codice/ddf/pull/2788**

#### What does this PR do?

Updates to the configuring section of the documentation; specifically configurations involving data management. 

#### Who is reviewing it? 

@mcalcote @vinamartin @ahoffer @garrettfreibott 

#### Choose 2 committers to review/merge the PR. 

@clockard
@coyotesqrl
@lessarderic
@pklinef
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

Review content. Most changes are just moving content, but some new material added.

#### What are the relevant tickets?
[DDF-3352](https://codice.atlassian.net/browse/DDF-3352)

#### Screenshots (if appropriate)

[documentation.pdf](https://github.com/codice/ddf/files/1604527/documentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_tables/MetacardValidityMarkerPlugin.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+:title: Metacard Validation Marker Plugin
+:id: ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin
+:type: table
+:status: published
+:application: ${ddf-catalog}
+:summary: Metacard Validation Filter Plugin.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_tables/MetacardValidityMarkerPlugin.adoc]: Filter?

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_tables/MetacardValidityMarkerPlugin.adoc]: fixed

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
rebased and rerunning build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**